main.py

```diff
@@ -1 +1,118 @@
 
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.common.keys import Keys
+import time
+
+class Testcase101:
+
+    def main(self):
+        driver = webdriver.Firefox(executable_path="C:\\Users\\Johny\\Downloads\\geckodriver-v0.33.0-win64\\geckodriver.exe")
+        driver.get("https://interview.supporthive.com/staff/")
+        driver.implicitly_wait(30)
+        driver.maximize_window()
+        driver.find_element(By.ID, "id_username").send_keys("Agent")
+        driver.find_element(By.ID, "id_password").send_keys("Agent@123")
+        driver.find_element(By.ID, "btn-submit").click()
+        tickets = driver.find_element(By.ID, "ember29")
+        action = ActionChains(driver)
+        action.move_to_element(tickets).perform()
+        statuses = driver.find_element(By.LINK_TEXT, "Statuses")
+        statuses.click()
+        driver.find_element(By.XPATH, "/html/body/div[3]/div/section/section/div/header/button").click()
+        driver.find_element(By.TAG_NAME, "input").send_keys("Issue Created")
+        statusColourSelect = driver.find_element(By.XPATH, "//div[@class='sp-replacer sp-light']")
+        statusColourSelect.click()
+        statusColourEnter = driver.find_element(By.XPATH, "//input[@class='sp-input']")
+        statusColourEnter.clear()
+        statusColourEnter.send_keys("#47963f")
+        r = Robot()
+        r.keyPress(KeyEvent.VK_ESCAPE)
+        firstElement = driver.find_element(By.XPATH, "//a[@id='first-link']")
+        firstElement.click()
+        secondElement = driver.find_element(By.XPATH, "//a[@id='second-link']")
+        secondElement.click()
+        driver.find_element(By.TAG_NAME, "textarea").send_keys("Status when a new ticket is created in HappyFox")
+        addCreate = driver.find_element(By.XPATH, "//button[@class ='hf-entity-footer_primary hf-primary-action ember-view']")
+        addCreate.click()
+        time.sleep(3)
```

Comment:
Instead of using sleep we can use explicit waits which will help in reduce unnecessary waiting time